.github/workflows/downstream.yml

```diff
@@ -51,6 +51,9 @@ jobs:
           - url: https://github.com/sparklemotion/mechanize
             name: mechanize
             command: "bundle exec rake test"
+          - url: https://github.com/stimulusreflex/stimulus_reflex
+            name: stimulus_reflex
+            command: "bundle exec rake test"
```

Comment:
In case it's causing any issues: you might want to only run `bundle exec rake ruby_test` since the default rake task also includes the JavaScript tests.